src/main/java/build/buildfarm/server/BuildFarmServer.java

```diff
@@ -65,6 +69,31 @@ public BuildFarmServer(String session, BuildFarmServerConfig config)
     this(session, ServerBuilder.forPort(config.getPort()), config);
   }
 
+  private Collection<String> getAuthTokens(BuildFarmServerConfig config) throws ConfigurationException {
+    Collection<String> tokens = new ArrayList<>();
+
+    if (config.getAuth().hasStatic()) {
+      tokens.addAll(config.getAuth().getStatic().getTokensList());
+
+      String tokensFile = config.getAuth().getStatic().getTokensFile();
```

Comment:
why does the token file name needs to be configurable ?
seems we can just put it's path into a constants. (it should be a k8s secret mounted to a known volume)